askama_parser/src/expr.rs

```diff
@@ -39,6 +39,10 @@ fn check_expr<'a>(
     allow_underscore: bool,
 ) -> Result<(), ParseErr<'a>> {
     match &expr.inner {
+        Expr::Var("crate") => Err(winnow::error::ErrMode::Cut(ErrorContext::new(
```

Comment:
"self", "super" and "Self" need to be rejected, too.

Reply:
`super` too.